src/styles/controls.less

```diff
@@ -0,0 +1,537 @@
+@import './base/animations.less';
+@import './base/helpers.less';
+@import './base/mixins.less';
+@import './base/reset.less';
+@import './base/settings.less';
+@import './base/structure.less';
+@import './base/typography.less';
+
+// Player Controls
+
+//Vars
+
+@size-player-controls-button-height: 15px;
+@size-player-controls-button-height-mini: 15px;
+
+//
+
+.media-control[data-media-control]
+.media-control-layer[data-controls] {
+  display: flex;
+  position: static;
+
+  button {
+    &:extend(.hover-opacity all);
+    background-repeat: no-repeat!important;
+    background-position: center center;
+    background-color: @color-grey!important;
+    mask-repeat: no-repeat;
+    mask-position: center;
+    -webkit-mask-repeat: no-repeat;
+    -webkit-mask-position: center;
+
+    &:hover {
+      background-color: @color-white!important;
+    }
+
+    .query-mini({
+      height: @size-player-controls-button-height-mini;
+    });
+
+    svg {
+      display: none;
+    }
+
+    &-volume {
+      margin: 0;
+      width: 15px;
+    }
+
+    &-fullscreen {
+      position: absolute;
+      right: 30px;
+      top: 30px;
+      transition: transform .8s @ease-smooth, opacity .3s;
+      width: @size-player-controls-button-height;
+      z-index: @zindex-player-button-fullscreen;
+
+      .query-mini({
+        height: 20px;
+        width: 20px;
+      });
+
+      .hide-controls & {
+        transform: translate3d(0, -60px, 0);
+        transition: transform .65s @ease-smooth, opacity .3s;
+      }
+
+      .button-fullscreen-svg {
+        fill: @color-white;
+      }
+    }
+  }
+
+
+  .media-control-left-panel[data-media-control] {
+    display: none;
+  }
+
+  .media-control-right-panel[data-media-control] {
+    display: none;
+  }
+
+  .media-control-center-panel[data-media-control] {
+    position: static!important;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    order: 1;
+    flex: 1 1 0;
+
+    .bar-container[data-seekbar] {
+      position: relative;
+      top: 0px;
+      display: flex;
+      align-items: center;
+
+      .bar-background[data-seekbar] {
+        height: 5px!important;
```

Comment:
There are a lot of `!important`s on this branch right now, but they will be removed. They are only here now to override styles in the default stylesheet, but eventually that stylesheet will be removed.